packages/o-spreadsheet-engine/src/plugins/core/cell.ts

```diff
@@ -537,6 +371,7 @@ export class CellPlugin extends CorePlugin<CoreState> implements CoreState {
     }
 
     const cellId = before?.id || this.getNextUid();
+    const format = this.getters.getCellFormat({ sheetId, col, row });
```

Comment:
There's a high chance this is a perf killer since `getCellFormat` is linear and `updateCell` can be called a huge number of time